samples/matter/manufacturer_specific/README.rst

```diff
@@ -119,6 +119,11 @@ Factory data support
     :start-after: matter_door_lock_sample_factory_data_start
     :end-before: matter_door_lock_sample_factory_data_end
 
+Registering code-driven clusters
+--------------------------------
+
+.. include:: /includes/matter_cluster_init.txt
```

Comment:
I'm wondering if we should include that guide this way. In fact now it applies to custom clusters and extensions, but in a near future it will apply to majority of clusters (including regular ones), so the use case is far wider than this sample. Maybe for the sake of future it could be added as a separate page to doc, or even grouped altogether with guides for adding new cluster, enabling new clusters. Moreover we currently have guide that shows how to create new cluster with Matter CLuster Editor and this sample, so maybe we should just add this guide there and link it here?